site/components/Button.js

```diff
@@ -7,7 +7,13 @@ const Button = function ({
   ...props
 }) {
   const boxStyle = `py-3 rounded-xl ${width} focus:outline-none`
-  const textStyle = 'text-center text-base text-white capitalize'
+  const textCase = className
+    .split(' ')
+    .filter(Boolean)
+    .some(c => ['lowercase', 'normal-case', 'uppercase'].includes(c))
+    ? ''
+    : 'capitalize'
+  const textStyle = `text-center text-base text-white ${textCase}`
```

Comment:
I don't really like this, could we consider using tailwind merge? if not now, let's create an issue to do so.